logp/core.go

```diff
@@ -151,6 +151,48 @@ func ConfigureWithOutputs(defaultLoggerCfg Config, outputs ...zapcore.Core) erro
 	return nil
 }
 
+// ConfigureWithCore configures the global logger to use the passed in
+// core.  It is assumed that an output has already been defined with
```

Comment:
[NIT]
Is there any reason to use double spaces after the period?

Reply:
I recall seeing this in some other PRs from Lee, maybe it is an editor configuration on his side?

Reply:
This is just for context, since you both seem interested.  I'm not getting into a flame war about white space formatting in comments.

It has to do with being old enough that I learned to type on an actual [typewriter](https://en.wikipedia.org/wiki/IBM_Selectric).  2 spaces between sentences were **required**.  To see why they were required take a look at a full sheet of text in Pica 10pt (Courier is pretty close), and make sure justification is disabled.  Without the 2 spaces it is very hard to scan text to see where one sentence ends and one begins.  And since you are using your eyes to "search", this is a very desirable feature.